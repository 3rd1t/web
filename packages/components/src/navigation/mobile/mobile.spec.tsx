import React from "react"
import { render } from "@testing-library/react"

import Mobile from "./mobile"

describe(" Mobile", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<Mobile logo={<span>Logo</span>} links={[{ label: "Hello", href: "/world" }]} />)
    expect(baseElement).toBeTruthy()
  })
})
