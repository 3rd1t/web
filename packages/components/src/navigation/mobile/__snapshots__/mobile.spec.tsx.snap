// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Mobile should render successfully 1`] = `
<body>
  <div>
    <div>
      <button
        aria-label="open menu"
        class="p-2 text-gray-700 hover:text-gray-900 focus:outline-none"
        data-testid="toggle"
      >
        <svg
          class="w-6 h-6"
          fill="none"
          stroke="currentColor"
          viewBox="0 0 24 24"
        >
          <path
            d="M4 6h16M4 12h16M4 18h16"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
          />
        </svg>
      </button>
    </div>
  </div>
</body>
`;
